exchanges/order/order_test.go

```diff
@@ -727,14 +609,10 @@ func TestSortOrdersByPrice(t *testing.T) {
 	}
 
 	SortOrdersByPrice(&orders, false)
-	if orders[0].Price != 0 {
-		t.Errorf("Expected: '%v', received: '%v'", 0, orders[0].Price)
-	}
+	assert.Equalf(t, float64(0), orders[0].Price, "Expected: '%v', received: '%v'", 0, orders[0].Price)
 
 	SortOrdersByPrice(&orders, true)
-	if orders[0].Price != 100 {
-		t.Errorf("Expected: '%v', received: '%v'", 100, orders[0].Price)
-	}
+	assert.Equalf(t, float64(100), orders[0].Price, "Expected: '%v', received: '%v'", 100, orders[0].Price)
```

Comment:
```suggestion
	assert.Equal(t, 100.0, orders[0].Price, "Price should be correct")
```

Since equalf is redudant with exp vs recv, the compiler is also smart enough to know 100.0 is a float with explicit float64 usage